web/src/modules/topic/components/Score/ScorePie.tsx

```diff
@@ -30,28 +43,33 @@ export const ScorePie = ({ pieDiameter, arguableId, arguableType }: Props) => {
   });
 
   return (
-    <Box display="flex" width={pieDiameter} height={pieDiameter}>
+    <StyledBox display="flex" width={pieDiameter} height={pieDiameter}>
       <PieChart
         data={data}
-        radius={50} // is relative to viewbox size [100, 100]
+        radius={45} // is relative to viewbox size [100, 100]
         label={({ dataEntry }) => dataEntry.key}
         labelStyle={() => ({
           fontSize: "14", // is relative to viewbox size [100, 100]
           pointerEvents: "none",
+          webKitUserSelect: "none" /* Safari */,
```

Comment:
hmm I'm looking at the css in the browser and I don't see the webkit css prefix. not sure what the issue is here, BUT our styling solution (emotion) actually automatically adds prefixes - it [uses stylis by default](https://emotion.sh/docs/@emotion/cache#stylisplugins) and you can check out what should be generated by typing css into stylis's [homepage](https://stylis.js.org/). in this case, i think you should be able to do something like
```
labelStyle={() => css({ [insert css here] ... })
```
and it should add the prefixes (see emotion docs for [css object styling](https://emotion.sh/docs/object-styles#with-css))

Reply:
Nice call out! i wasn't familiar with the syntax `labelStyle={() => css({ [insert css here] ... })`
works like a charm <3
`transform: dataIndex === hovered ? "scale(1.1)" : "",`